lib/services/file.dart

```diff
@@ -40,6 +40,55 @@ Future<XFile?> cropImage(
   );
 }
 
+Completer<SnCloudFile?> putFileToPool({
```

Comment:
Instead of creating a new method, use a optional argument might be a better way to handle this situation